Project_Organization.md

```diff
@@ -62,6 +62,18 @@ Working Group meet their goals, and any other activities Approved by the
 Steering Committee. The Chairperson may delegate responsibilities to another
 party. The Steering Committee may Approve a new Chairperson.
 
+### 3.3. Rhythm of Business
+
+The seat of a Steering Member may be vacated if a Steering Committee delegate
+is not designated or absent for more than one third of Steering Committee meetings
+in one year without the prior consent of the Steering Committee. Prior consent
```

Comment:
Should we allow absent SC members to nominate stand-ins from their organisation to cover for them?  I'd be in favour of this.